packages/e2e/support/hooks.ts

```diff
@@ -43,3 +50,7 @@ After(async function(scenario) {
     process.exit(1);
   }
 });
+
+AfterAll(async function() {
+  await destroyTestUser(testUser);
```

Comment:
I mentioned before that we need to think something of the error handling and how it would behave.. I will try to think of a proposal for this at some point, but just something to keep in mind